pkgs/development/python-modules/pandas/default.nix

```diff
@@ -8,37 +9,43 @@
 , cython
 , dateutil
 , html5lib
+, jinja2
 , lxml
 , numexpr
 , openpyxl
 , pytz
-, sqlalchemy
 , scipy
+, sqlalchemy
 , tables
+, XlsxWriter
 , xlrd
 , xlwt
-# Test Inputs
+# Test inputs
 , glibcLocales
 , hypothesis
 , pytestCheckHook
+, pytest-xdist
+, pytest-asyncio
 # Darwin inputs
 , runtimeShell
-, libcxx ? null
+, libcxx
 }:
 
 buildPythonPackage rec {
   pname = "pandas";
-  version = "1.1.5";
+  version = "1.2.2";
```

Comment:
1.2.3 was released yesterday

Reply:
@fabaff As this is blocking staging-next, can you prioritize updating this?

Reply:
Sure